File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-if.js
const items = loadItems();
let index = 0;
while (index < items.length) {
    const item = items[index];
    if (item.enabled) {
        processItem(item);
    } else {
        recordSkippedItem(item);
    }
    index = index + 1;
}
showSummary();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-if.java
List<Item> items = loadItems();
int index = 0;
while (index < items.size()) {
    Item item = items.get(index);
    if (item.isEnabled()) {
        processItem(item);
    } else {
        recordSkippedItem(item);
    }
    index = index + 1;
}
showSummary();
